dev/io.openliberty.restfulWS.3.1_fat/fat/src/io/openliberty/restfulWS31/fat/multipart/MultipartTestServlet.java

```diff
@@ -126,14 +126,14 @@ public void testMultipartResponse() throws Exception {
             assertEquals("Value4", header2ValuesArr[2].strip());
         } else if (header2ValuesList.size() == 3) {
             // RESTEasy returns a list of 3 strings
-            assertEquals("[Value2]", header2ValuesList.get(0));
-            assertEquals("[Value3]", header2ValuesList.get(1));
-            assertEquals("[Value4]", header2ValuesList.get(2));
+            assertEquals("Value2", header2ValuesList.get(0));
+            assertEquals("Value3", header2ValuesList.get(1));
+            assertEquals("Value4", header2ValuesList.get(2));
         } else {
             fail("unexpected number of header values for Header2");
```

Comment:
```suggestion
            fail("Unexpected number of header values for Header2: " + header2ValuesList.toString());
```

To match the change you did in the other test class.

Reply:
agreed